src/components/backend-ai-resource-policy-list.ts

```diff
@@ -641,21 +633,27 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
     this._validateUserInput(this.ram_resource);
     this._validateUserInput(this.gpu_resource);
     this._validateUserInput(this.fgpu_resource);
+
+    // Integer (4byte)
     this._validateUserInput(this.concurrency_limit);
-    this._validateUserInput(this.idle_timeout);
     this._validateUserInput(this.container_per_session_limit);
-    this._validateUserInput(this.vfolder_capacity);
     this._validateUserInput(this.vfolder_max_limit);
 
+    // BigInt
+    this._validateUserInput(this.vfolder_capacity);
+    this._validateUserInput(this.idle_timeout);
+
     total_resource_slots['cpu'] = this.cpu_resource['value'];
     total_resource_slots['mem'] = this.ram_resource['value'] + 'g';
     total_resource_slots['cuda.device'] = parseInt(this.gpu_resource['value']);
     total_resource_slots['cuda.shares'] = parseFloat(this.fgpu_resource['value']);
-    this.concurrency_limit['value'] = ['', 0].includes(this.concurrency_limit['value']) ? BigNumber.getValue() : parseInt(this.concurrency_limit['value']);
-    this.idle_timeout['value'] = this.idle_timeout['value'] === '' ? 0 : parseInt(this.idle_timeout['value']);
-    this.container_per_session_limit['value'] = ['', 0].includes(this.container_per_session_limit['value']) ? BigNumber.getValue() : parseInt(this.container_per_session_limit['value']);
-    this.vfolder_capacity['value'] = this.vfolder_capacity['value'] === '' ? 0 : parseFloat(this.vfolder_capacity['value']);
-    this.vfolder_max_limit['value'] = this.vfolder_max_limit['value'] === '' ? 0 : parseInt(this.vfolder_max_limit['value']);
+
+    this.concurrency_limit['value'] = ['', 0].includes(this.concurrency_limit['value']) ? BackendAIResourcePolicyList.MAX_INT32 : parseInt(this.concurrency_limit['value']);
+    this.container_per_session_limit['value'] = ['', 0].includes(this.container_per_session_limit['value']) ? BackendAIResourcePolicyList.MAX_INT32 : parseInt(this.container_per_session_limit['value']);
+    this.vfolder_max_limit['value'] = (this.vfolder_max_limit['value'] === '') ? 0 : parseInt(this.vfolder_max_limit['value']);
+
+    this.idle_timeout['value'] = (this.idle_timeout['value'] === '') ? Number.MAX_SAFE_INTEGER : parseInt(this.idle_timeout['value']);
```

Comment:
Is there a reason to change the default from `0` to `MAX_SAFE_INTEGER` when the value is empty (`''`)? Practically, they may be the same, but I am curious.

Reply:
Right, it's weird. I'll just change it to zero since it means infinity. (https://github.com/lablup/backend.ai/blob/141844d383970b7274b9861b625d7f51a6601998/src/ai/backend/manager/models/resource_policy.py#L55)